src/core/constants.js

```diff
@@ -323,3 +323,5 @@ export const LTR = 'ltr';
 
 export const AMO_REQUEST_ID_HEADER = 'amo-request-id';
 export const DISCO_TAAR_CLIENT_ID_HEADER = 'moz-client-id';
+
+export const USER_AGENT_BROWSER_FIREFOX = 'Firefox';
```

Comment:
I thought about putting it there, because the other constants that start with `USER_AGENT` are in there, but it's not used by any API-related code, so I figured it made more sense to just put it into `core/constants` like most of our other constants. I can certainly move it if you like.

Reply:
I moved it.